src/test/java/features/StartingWorldSteps.java

```diff
@@ -0,0 +1,38 @@
+package features;
+
+import github.workflow.kata.GOL;
+import io.cucumber.java8.En;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.nio.charset.StandardCharsets;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StartingWorldSteps implements En {
+    private final ByteArrayOutputStream appResultOutputStream = new ByteArrayOutputStream();
+
+    /**
+     * Creates a new StartingWorldSteps, with the code to make the starting world feature work.
+     */
+    public StartingWorldSteps() {
+        /*
+        Starts the application flushing the definition into standard input.
+         */
+        Given("a world initialized from the input", (String definition) -> {
+            String input = definition + "\n\n";
+            InputStream customin = new ByteArrayInputStream(input.getBytes(StandardCharsets.UTF_8));
+            InputStream oldIn = System.in;
+            System.setIn(customin);
+            System.setOut(new PrintStream(appResultOutputStream));
+            GOL.main(new String[]{});
```

Comment:
Oh boy, really? Globals are bad you know.

Reply:
I think I would have made the "game" a class that can be run. and then the main class is simply a thin bootstrap to deliver props/args to the game class. Dont make the game class handle stdin/stdout please.

Reply:
I will leave it for now, will add an issue that will make the architecture a bit more sound:
GOL - starter for the game, handles input/output/help etc. Test covered by a simple set of BDD tests. (with the 
 global var abdomination)
Grid - Actual game class, covered by more robust BDD and Junit tests.